RELEASE.md

```diff
@@ -117,6 +126,14 @@ This parts works only inside the Red Hat network.
        $ rhpkg push
        $ rhpkg build
 
+   If the push command fails, make sure the correct public key algorithm
+   is used.
+
+       $ cat ~/.ssh/config
+       Host pkgs.devel.redhat.com
+          HostkeyAlgorithms +ssh-rsa
+          PubkeyAcceptedAlgorithms +ssh-rsa
```

Comment:
This is also private information that is not good for public document. You can keep it
if you remove the actual server address, or if you find the public page documenting this
issue. If you don't find a public doc for this issue, we can keep it.

Reply:
I think I found both links :)